src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBUtils.java

```diff
@@ -36,6 +37,50 @@
     private static final String STRAND_BIAS_TABLE_COMBINE = "strand_bias_table";
```

Comment:
@nalinigans -- yes, changing this to ELEMENT_WISE_INT_SUM is a great idea!

Reply:
Done.